build-jekyll-tree.sh

```diff
@@ -33,7 +33,32 @@ function main {
   mkdir -p "$OUT_DIR"
   cd "$OUT_DIR"
   tar -xf "${JEKYLL_BASE}"
+
+  if [[ -n "$REDIRECTS_FILE" ]]; then
+    local redirects_path="${OWD}/${REDIRECTS_FILE}"
+    while IFS=$'\t' read old new; do
+      if [[ -e "$old" ]]; then
+        echo "Cannot create redirect file $old. File exists."
+        exit 1
+      fi
+      local old_dir=$(dirname $old)
+      if [[ ! -d "$old_dir" ]]; then
+        mkdir -p "$old_dir"
+      fi
+      cat <<EOF > $old
+<html>
+  <head>
+    <script>
+      window.location.replace("${new}" + window.location.hash);
```

Comment:
Is JavaScript the only way to achieve a redirect? Ideally we would be using HTTP redirects.